pkg/ensurance/collector/nodelocal/load.go

```diff
@@ -0,0 +1,58 @@
+package nodelocal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/shirou/gopsutil/load"
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/common"
+	"github.com/gocrane/crane/pkg/ensurance/collector/types"
+)
+
+const (
+	loadCollectorName = "load"
```

Comment:
is this cpuLoad? if yes, the file should be renamed to cpuload.go
and all variables should follow

Reply:
it cpu load , I rename all